docs/developer-guide/coal-setup.md

```diff
@@ -313,8 +313,7 @@ global                               running         smartlogin          online
     adminui0 10.99.99.31 10.88.88.3
     ```
 
-   We can now access access the operations portal, "SDC ADMINUI", in a web
-   browser on the host computer at https://10.88.88.3/ .
+   You can now access the Operations Portal (SDC ADMINUI) in a web browser on the host computer at <https://10.88.88.3>.
```

Comment:
All prose should be hard wrapped prior to 80 columns for readability in terminals.